scripts/makefile/patchval.sh

```diff
@@ -1,26 +1,15 @@
 #!/usr/bin/env sh
 # set -x
 
-# Get name of parent branch...
-if [ -z "$CI_MERGE_REQUEST_TARGET_BRANCH_NAME" ]; then
-  # ... when not in Gitlab CI
-  PARENT_BRANCH=$(git show-branch | sed "s/].*//" | grep "\*" | grep -v "$(git rev-parse --abbrev-ref HEAD)" | head -n1 | sed "s/^.*\[//")
-else
-  # ... when in Gitlab CI
-  PARENT_BRANCH=$CI_MERGE_REQUEST_TARGET_BRANCH_NAME
-fi
-
-# Diff current branch against parent branch and make validation
-git fetch -q origin "$PARENT_BRANCH"
-PATCH_FILE_COUNT=$(git diff origin/"$PARENT_BRANCH" --name-status --diff-filter=A | grep -c "\.patch")
+NON_REMOTE_PATCH_COUNT=$(jq --raw-output '.extra.patches[] | .[]' composer.json | grep -v "http" | wc -l)
```

Comment:
does running container has `jq` ?

Reply:
We make sure of that by installing it silently in `make patchval` anyways : https://github.com/skilld-labs/skilld-docker-container/pull/356/files#diff-60293e389da11b5b5f133a3a0f2f32e1901fc50e61edc6d0aca6bd6216b6e4daR138